ouroboros-consensus/changelog.d/20240808_101916_marcin.wojtowicz_tx_wire_size.md

```diff
@@ -0,0 +1,23 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+<!--
+### Non-Breaking
+
+- A bullet item for the Non-Breaking category.
+
+-->
+### Breaking
+
+* Added txWireSize method to LedgerSupportsMempool to provide
```

Comment:
I think it doesn't matter but we use `-` not `*`

Reply:
As it goes, when in Rome... :)